umbral/keys.py

```diff
@@ -82,8 +79,8 @@ def from_bytes(cls, key_bytes: bytes, params: UmbralParameters=None,
         bn_key = CurveBN.from_bytes(key_bytes, params.curve)
         return cls(bn_key, params)
 
-    def to_bytes(self, password: bytes=None, _scrypt_cost: int=20,
-                 encoder: Callable=None):
+    def to_bytes(self, password: bytes = None, _scrypt_cost: int = 20,
```

Comment:
In instances where you set a default value with type in the function like the `_scrypt_cost` variable here, the equal sign should have no spaces such that:

`_scrypt_cost: int=20`

In cases where there is no type, such as `None`, it should have a space:

`password: bytes = None`

https://www.python.org/dev/peps/pep-0008/#other-recommendations

I'm not necessarily a huge fan of this PEP8 rule, as such I will leave it up to everyone else to decide how to do this. Generally, I'm in favor of not doing this as it just isn't necessary.

Reply:
I used to think this was a really dumb style, but then I came around to it.  The thing is: in kwargs, `x=y` looks like assignment.  With the space around it, your eyes learn to treat the space as a type hint instead of argument assignment.

Reply:
🤔 But it's not a type hint? It's a default assignment if the argument is not passed?

Reply:
Eh?  It is a type hint.  Without the space, it looks like you're assigning the value `None` to the name `bytes`.

Reply:
Oh, I see! Gotcha.